src/models/chat/index.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * The action types accepted by the Analytics Events API.
+ *
+ * @public
+ */
+export const ACTIONS = [
```

Comment:
I think this can just be `Actions`

Reply:
Yep that works.

Reply:
[Made this into a string literal per our conversation in Slack.](https://yext.slack.com/archives/C04EW3KMLEL/p1687800787175749)